src/DynamoCore/Logging/DynamoAnalyticsClient.cs

```diff
@@ -213,7 +211,7 @@ public DynamoAnalyticsClient(DynamoModel dynamoModel)
                 buildId = $"{version.Major}.{version.Minor}.{version.Build}"; // BuildId has the following format major.minor.build, ex: 2.5.1
                 releaseId = $"{version.Major}.{version.Minor}.0"; // ReleaseId has the following format: major.minor.0; ex: 2.5.0
             }
-            product = new ProductInfo() { Id = "DYN", Name = hostName, VersionString = appversion, AppVersion = appversion, BuildId = buildId, ReleaseId = releaseId };
+            product = new ProductInfo() { Id = "DYN", Name = "Dynamo", VersionString = appversion, AppVersion = appversion, BuildId = buildId, ReleaseId = releaseId };
```

Comment:
hmm Why did we revert the change here?

Reply:
I can switch back to hostName. I was waiting for your go ahead.